<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CBatchSetCardAccess</class>
 <widget class="QFrame" name="CBatchSetCardAccess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1390</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>批量设置卡片进出权</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(D:/WinParkUI/debug/Image/NewIcon/CommonBigBG-normal.jpg);</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>1301</width>
     <height>731</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>0</y>
      <width>301</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <widget class="QTableWidget" name="tableBatchChannel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>281</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <column>
      <property name="text">
       <string>通道名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>选择</string>
      </property>
     </column>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>510</y>
       <width>281</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <property name="title">
      <string>时间段设置</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>86</width>
        <height>106</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(none);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTimeEdit" name="timeStart">
         <property name="styleSheet">
          <string notr="true">background-image:url(none);</string>
         </property>
         <property name="displayFormat">
          <string>HH:mm:ss</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="timeEnd">
         <property name="styleSheet">
          <string notr="true">background-image:url(none);</string>
         </property>
         <property name="dateTime">
          <datetime>
           <hour>23</hour>
           <minute>59</minute>
           <second>59</second>
           <year>2000</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="displayFormat">
          <string>HH:mm:ss</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAllTime">
         <property name="styleSheet">
          <string notr="true">background-image:url(none);</string>
         </property>
         <property name="text">
          <string>全时段</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnForbidden">
         <property name="styleSheet">
          <string notr="true">background-image:url(none);</string>
         </property>
         <property name="text">
          <string>禁止通过</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>56</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(none);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">background-image:url(none);</string>
         </property>
         <property name="text">
          <string>起始：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">background-image:url(none);</string>
         </property>
         <property name="text">
          <string>终止：</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btnAllUnselected_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>102</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(none);</string>
      </property>
      <property name="text">
       <string>全取消</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAllSecleted_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>73</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(none);</string>
      </property>
      <property name="text">
       <string>全选中</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>月租卡</string>
     </attribute>
     <widget class="QTableWidget" name="tableBatchMonth">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>971</width>
        <height>621</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(none);</string>
      </property>
      <column>
       <property name="text">
        <string>卡号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>自编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>用户名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>选择</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>计时卡</string>
     </attribute>
     <widget class="QTableWidget" name="tableBatchTime">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>971</width>
        <height>621</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>卡号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>自编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>用户名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>选择</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>储值卡</string>
     </attribute>
     <widget class="QTableWidget" name="tableBatchValue">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>971</width>
        <height>621</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>卡号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>自编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>用户名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>选择</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>680</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <widget class="QPushButton" name="btnAllUnselected">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <property name="text">
      <string>全取消</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAllSecleted">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <property name="text">
      <string>全选中</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMinimalize">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>0</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(none);</string>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnOk">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>680</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="text">
     <string>确认修改</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>5</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);
color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblClose">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>7</y>
     <width>30</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
